pkg/lib/accountmanagement/service.go

```diff
@@ -184,3 +277,220 @@ func (s *Service) FinishAdding(input *FinishAddingInput) (*FinishAddingOutput, e
 
 	return &FinishAddingOutput{}, nil
 }
+
+type ResendOTPCodeInput struct {
+	UserID       string
```

Comment:
It should not take UserID. It should take token. ResendOTPCode should be in the middleware of a flow.